listenbrainz/db/lovehate.py

```diff
@@ -0,0 +1,138 @@
+import sqlalchemy
+
+from listenbrainz import db
+
+
+def insert_lovehate_record(user_id, recording_msid, score):
+    """ Inserts a record for a user's loved/hated recording into the database.
+        If the record is already present for the user, the score is updated to the new
+        value passed.
+        Args:
+            user_id (int): the row id of the user
+            recording_msid (string): the MessyBrainz ID of the recording
+            score (int): The score associated with the recording (+1/-1 for love/hate respectively)
+    """
+
+    with db.engine.connect() as connection:
+        connection.execute(sqlalchemy.text("""
+            INSERT INTO lovehate (user_id, recording_msid, score)
+                 VALUES (:user_id, :recording_msid, :score)
+            ON CONFLICT (user_id, recording_msid)
+          DO UPDATE SET score = :score,
+                        created = NOW()
+            """), {
+                'user_id': user_id,
+                'recording_msid': recording_msid,
+                'score': score,
+            }
+        )
+
+
+def delete_lovehate_record(user_id, recording_msid):
+    """ Deletes the record for a user's loved/hated recording from the database.
+        Args:
+            user_id (int): the row id of the user
+            recording_msid (dict): the MessyBrainz ID of the recording
+    """
+
+    with db.engine.connect() as connection:
+        connection.execute(sqlalchemy.text("""
+            DELETE FROM lovehate
+             WHERE user_id = :user_id
+               AND recording_msid = :recording_msid
+            """), {
+                'user_id': user_id,
+                'recording_msid': recording_msid,
+            }
+        )
+
+
+def get_user_loved_or_hated_recordings(user_id, score):
```

Comment:
get_feedback() ?

Reply:
This function and the next feel weird to me -- not sure we ever want to fetch just one. Buy ok, it exists... But the names are confusing -- you have to read the docs before you get what they mean. 

@paramsingh what do you think?

Reply:
I tried to resolve the names to better names. Perhaps these will be good now

Reply:
Added better names